ifrs17/Utils/ActivityLog.ipynb

```diff
@@ -157,6 +157,11 @@
                 "\n    public string Format {get; init;}",
                 "\n    protected IDataSetImportVariable DataSetReader {get; set;}",
                 "\n    protected ISessionVariable Session {get; set;}",
+                "\n",
+                "\n    public KeyedImportExport() ",
```

Comment:
what is this?

Reply:
This is actually a default constructor that will be used without change by the inheriting classes.